frontend/app/views/spree/checkout/_payment.html.erb

```diff
@@ -8,12 +8,12 @@
     <%= render partial: 'spree/checkout/payment/storecredit' %>
 
     <ul id="payment-method-fields" class="list-unstyled position-relative" data-hook>
-      <% @order.available_payment_methods.each do |method| %>
+      <% @order.available_payment_methods(current_store).each do |method| %>
```

Comment:
@kaliszKrzysztof let's move this into `checkout_available_payment_methods` method in `FrontendHelper` please